addons/mail/static/src/js/activity.js

```diff
@@ -543,6 +543,7 @@ var BasicActivity = AbstractField.extend({
 // -----------------------------------------------------------------------------
 // Activities Widget for Form views ('mail_activity' widget)
 // -----------------------------------------------------------------------------
+// FIXME seems to still be needed in some cases like systray
```

Comment:
Didn't look into it very deeply, I just understood that removing Activity class makes it fail so I brought the class back. Don't know if it could be easy to rewrite or not